src/index.ts

```diff
@@ -36,6 +37,26 @@ export async function main() {
     )
     core.endGroup()
 
+    core.startGroup("Load json files into context")
+    const files = core.getInput("json-files-to-include")
+    if (files !== "") {
+      const filesJson = JSON.parse(files)
+      // loop through and load each file into context with the name and files json contents
+      for (const file of filesJson) {
+        const name = file.name
+        const filepath = file.path
+        try {
+          const data = fs.readFileSync(filepath, "utf8")
+          const jsonData = JSON.parse(data)
```

Comment:
Is there an expected format here? If so, we should probably use `zod` again. It's a lot easier to validate the data in here and reject on a mismatch than it is to clean up the uploaded logs

Reply:
I was trying to leave this format open here so we could have one function to upload test results data and not need to know what the format is since it will be different depending on the language. If enforcing it here is the better idea then I will remove my go github action that does the formatting enforcement and convert it from jq to typescript.

Reply:
Yeah I think it'll be better to have the format strictly typed, otherwise it will make the loki queries really hard.